org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/ClasspathShortener.java

```diff
@@ -313,6 +317,25 @@ private void shortenPathUsingClasspathArgumentFile(int modulePathArgumentIndex,
 		addProcessTempFile(file);
 	}
 
+	private void createOrUpdateGitIgnore() {
+		File gitIgnore = new File(processTempFilesDir, ".gitignore"); //$NON-NLS-1$
```

Comment:
Instead of touching gitignore, one can contribute something like:
```
  <extension
        point="org.eclipse.team.core.ignore">
     <ignore
           enabled="true"
           pattern=".temp-*path-arg-*.txt"/>
  </extension>
```

Reply:
Unfortunately neither ".temp-*path-arg-*.txt" nor "**/.temp-*path-arg-*.txt" actually hides these files in the Package Explorer; I guess I have to investigate further.

Reply:
> would it be feasible to use a dedicated temp path below
workspace\.metadata\.plugins\org.eclipse.jdt.launching?

Technically we can create these temp files there, yes. I wonder why that wasn't done and if there is a catch (like having them in the base directory etc).